package.json

```diff
@@ -3,37 +3,53 @@
   "version": "7.0.0-beta.0",
   "description": "The router for easy microfrontends",
   "type": "module",
-  "main": "lib/umd/single-spa.min.cjs",
+  "main": "./lib/umd/single-spa.min.cjs",
+  "types": "./lib/umd/types/single-spa.d.cts",
+  "module": "./lib/esm/single-spa.min.js",
```

Comment:
I don't think there are bundlers still in use that would alter there behavior because of the presence of this field, since there is a `"type"` and a `"main"` field. The link you shared from esbuild suggests switching to the `"browser"` field instead. Let's remove this

```suggestion
```

Reply:
From my research, this field is still used by rollup, and webpack.
From rollup docs:

> If your package.json file also has a module field, ES-module-aware tools like Rollup and [webpack 2+](https://webpack.js.org/) will [import the ES module version](https://github.com/rollup/rollup/wiki/pkg.module) directly.

Webpack has a configuration resolve.mainFields which defaults to (in order of resolution): ['browser', 'module', 'main']

Let me know if you prefer to have it removed.

Reply:
It should be removed, I'll push a commit